Update GBC 2023 Autumn wiki page (ZH+EN) (#10336)
ppy/osu-wiki: wiki/Tournaments/GBC/GBC_2023_Autumn/en.md

## Prizes

> ### QQ Division

Reviewer: ```suggestion
### QQ division
```

| Team Placing | Prizes |
| :-: | :-- |
| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | Profile badge, KanonBot profile badge, ¥150 CNY |
| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | KanonBot profile badge, ¥150 CNY |
| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | KanonBot profile badge, ¥120 CNY |
| *4th place* | ¥90 CNY |
| *5–6th place* | ¥75 CNY |
| *7–8th place* | ¥60 CNY |
| *9–12th place* | ¥45 CNY |
| *13–16st place* | ¥30 CNY |

> | Every Winning Match in Regular Stage | ¥50 CNY |

Reviewer: what does "Every Winning Match in Regular Stage" this mean? it doesn't make sense rn
it should also be in sentence case

Reviewer: ```suggestion
| *For every match won in the regular stage* | ¥50 CNY |
```

> | Qualifier Placing | Prizes |

Reviewer: ```suggestion
| Qualifier placing | Prizes |
```

| :-: | :-- |
| *1st place* | ¥200 CNY |
| *2nd place* | ¥150 CNY |
| *3rd place* | ¥100 CNY |
| *4-8th place* | ¥75 CNY |
| *9–24th place* | ¥50 CNY |
| *All the other qualified players* | ¥20 CNY |

> ### Discord Division

Reviewer: ```suggestion
### Discord division
```

> | Team Placing | Prizes |

Reviewer: ```suggestion
| Team placing | Prizes |
```

| :-: | :-- |
| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | Profile badge, ¥150 CNY |
| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | ¥150 CNY |
| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | ¥120 CNY |
| *4th place* | ¥90 CNY |
| *5–6th place* | ¥75 CNY |
| *7–8th place* | ¥60 CNY |
| *9–12th place* | ¥45 CNY |
| *13–16st place* | ¥30 CNY |

> | Every Winning Match in Regular Stage | 6$ |

Reviewer 2: ```suggestion
| Every Winning Match in Regular Stage | $6 |
```

> | Every Winning Match in Regular Stage | $6 |

Reviewer: same as above

Reviewer: ```suggestion
| *For every match won in the regular stage* | $6 |
```

> | Qualifier Placing | Prizes |

Reviewer: ```suggestion
| Qualifier placing | Prizes |
```

| :-: | :-- |
| *1st place* | $16 |
| *2nd place* | $13 |
| *3rd place* | $10 |

> | *4-8th place* | $8 |

Reviewer: ```suggestion
| *4–8th place* | $8 |
```

> | *1st place* | 16$ |
> | *2nd place* | 13$ |
> | *3rd place* | 10$ |
> | *4-8th place* | 8$ |
> | *9–16th place* | 6$ |
> | *All the other qualified players* | 2$ |

Reviewer 2: ```suggestion
| *1st place* | $16 |
| *2nd place* | $13 |
| *3rd place* | $10 |
| *4-8th place* | $8 |
| *9–16th place* | $6 |
| *All the other qualified players* | $2 |
```

## Organisation

> The GB Cup 2022 Autumn was run by the Team GB and various community members.

Reviewer 2: ```suggestion
The GB Cup 2023 Autumn was run by the Team GB and various community members.
```

> The GB Cup 2023 Autumn was run by the Team GB and various community members.

Reviewer: why is this in past tense now? isn't the tournament still ongoing?

| Position | Member(s) |
| :-- | :-- |
| Host | ::{ flag=CN }:: [\[GB\]yobrevelc](https://osu.ppy.sh/users/14128407) |
| Staff | ::{ flag=KR }:: [\[GB\]SuddenDeath](https://osu.ppy.sh/users/6699923), ::{ flag=CN }:: [\[GB\]Mafufu](https://osu.ppy.sh/users/10884561), ::{ flag=CN }:: [\[GB\]GanyuAngel](https://osu.ppy.sh/users/27233308) |
| Mappool selector | ::{ flag=CN }:: [\[GB\]yobrevelc](https://osu.ppy.sh/users/14128407), ::{ flag=CN }:: [\[GB\]Thaumiel](https://osu.ppy.sh/users/15017771), ::{ flag=CN }:: [\[GB\]gluefly](https://osu.ppy.sh/users/8715409), ::{ flag=CN }:: [\[GB\]DiGua](https://osu.ppy.sh/users/25308131) |

> | SV Mappool selector | ::{ flag=CN }:: [\[GB\]Mafufu](https://osu.ppy.sh/users/10884561), ::{ flag=VE }:: [Lott](https://osu.ppy.sh/users/13821222) |
> | Custom Mapper | *TBA* |

Reviewer: ```suggestion
| SV mappool selector | ::{ flag=CN }:: [\[GB\]Mafufu](https://osu.ppy.sh/users/10884561), ::{ flag=VE }:: [Lott](https://osu.ppy.sh/users/13821222) |
| Custom mapper | *TBA* |
```

- [GBC Discord server](https://discord.gg/kR4hJnPs)
- [Information spreadsheet](https://docs.qq.com/sheet/DTVlRcExwbUpTVVZn?tab=BB08J2)

## Participants

> ### QQ Division

Reviewer: ```suggestion
### QQ division
```

| Q22 | **和对面凑一队** | ::{ flag=CN }:: **[Wings_AnChen](https://osu.ppy.sh/users/12694894)**, ::{ flag=CN }:: [ATH_1gr](https://osu.ppy.sh/users/11931583), ::{ flag=CN }:: [xibuniuzai](https://osu.ppy.sh/users/10513110), ::{ flag=CN }:: [Apr4th](https://osu.ppy.sh/users/20485400), ::{ flag=CN }:: [neeeeeh](https://osu.ppy.sh/users/18586390) |
| Q23 | **再奖励一把大天空** | ::{ flag=CN }:: **[Yurucawaii](https://osu.ppy.sh/users/25682415)**, ::{ flag=CN }:: [- Xiaoluoli -](https://osu.ppy.sh/users/9502281), ::{ flag=CN }:: [Tempest_WH](https://osu.ppy.sh/users/16039216), ::{ flag=CN }:: [CLLbin](https://osu.ppy.sh/users/11837219), ::{ flag=CN }:: [mayoi123](https://osu.ppy.sh/users/25379764) |
| Q24 | **猫猫统治世界** | ::{ flag=CN }:: **[Carpihat](https://osu.ppy.sh/users/10085090)**, ::{ flag=CN }:: [Akermo](https://osu.ppy.sh/users/14934242), ::{ flag=CN }:: [\[GB\]Azukisan](https://osu.ppy.sh/users/5378805), ::{ flag=CN }:: [superSSS](https://osu.ppy.sh/users/4315477), ::{ flag=CN }:: [\[GB\] limata](https://osu.ppy.sh/users/21326517) |

> ### Discord Division

Reviewer: ```suggestion
### Discord division
```

### Tournament registration

1. Please send the host a **private message** and **fill [this form](https://wj.qq.com/s2/13179234/c366/)** to register. 

>    - Players need to choose a division according to their chat software, i.e. the Discord division or QQ division. Both the QQ division and Discord division will have an independent tournament, but use same mapppool.

Reviewer: ```suggestion
   - Players need to choose a division according to their chat software, i.e. the Discord division or QQ division. The QQ division and Discord division will each have an independent tournament, but use same mappool.
```

1. Following the Qualifier stage, all players will participate in a phase to form teams, called the *drafting phase*. Both divisions will have independent drafting phases. All unpicked players will be eliminated from the tournament.
2. Following the drafting phase, a Swiss stage will be played in both zones, called the *regular stage*. 16 teams will advance in total, and the others get eliminated from the tournament.

> 3. Following the regular stage, the elimination stage will be played in both zones, called *playoffs*. The top 8 teams from the regular stage will advance to a winner bracket, other teams will get into loser bracket.

Reviewer: ```suggestion
3. Following the regular stage, the elimination stage will be played in both divisions, called *playoffs*. The top 8 teams from the regular stage will advance to a winner bracket, other teams will get into loser bracket.
```

### Regular phase instructions

> 1. The regular phase will be divided into two division, the Discord division and the QQ division.

Reviewer: ```suggestion
1. The regular phase will be divided into two divisions, the Discord division and the QQ division.
```
